subprojects/docs/src/docs/userguide/custom_tasks.adoc

```diff
@@ -47,23 +47,19 @@ In our examples, we will start with the task class in the build script, to keep
 
 To implement a custom task class, you extend link:{groovyDslPath}/org.gradle.api.DefaultTask.html[DefaultTask].
```

Comment:
Note: I think it would be nice to explain why the task class must be open in Kotlin, but I don't know why (I understand a proxy is created based on the error message, but I don't know what it's used for)

Reply:
The task class must not be final because Gradle will decorate it at instantiation. In Groovy it means you can't make it `final`. In Kotlin it means you must make it `open`.